docs/Users_Guide/release-notes.rst

```diff
@@ -47,359 +47,53 @@ When applicable, release notes are followed by the
-       (`#1667 <https://github.com/dtcenter/METplus/issues/1667>`_)
-     * Specify available tags on DockerHub
-       (`#2329 <https://github.com/dtcenter/METplus/issues/2329>`_)
-
-  .. dropdown:: Internal
-
-     * Improve METplus test coverage
-       (`#2253 <https://github.com/dtcenter/METplus/issues/2253>`_)
-     * Documentation: Make Headers Consistent in METplus components User's Guides
-       (`#898 <https://github.com/dtcenter/METplus/issues/898>`_)
-
-METplus Version 6.0.0 Beta 1 Release Notes (2023-09-15)
--------------------------------------------------------
-
-  .. dropdown:: Enhancements
-
-     * **Remove support for deprecated environment variables for old wrapped MET config files**
-       (`#2299 <https://github.com/dtcenter/METplus/issues/2299>`_)
-     * Improve time formatting logic to include certain times and use day of week to subset
-       (`#2283 <https://github.com/dtcenter/METplus/issues/2283>`_)
-     * Remove TCMPRPlotter wrapper
-       (`#2310 <https://github.com/dtcenter/METplus/issues/2310>`_)
-
-  .. dropdown:: Bugfix
-
-     * Update buoy use case to use buoy station file from 2022
-       (`#2279 <https://github.com/dtcenter/METplus/issues/2279>`_)
-     * Prevent failure in LSR use case
-       (`#2294 <https://github.com/dtcenter/METplus/issues/2294>`_)
-
-
-  .. dropdown:: New Wrappers
-
-     NONE
-
-  .. dropdown:: New Use Cases
-
-     * Scatterometer wind data
-       (`#1488 <https://github.com/dtcenter/METplus/issues/1488>`_)
-
-  .. dropdown:: Documentation
-
-     NONE
-
-  .. dropdown:: Internal
-
-     * Add coordinated release checklist to the METplus Release Guide
-       (`#2282 <https://github.com/dtcenter/METplus/issues/2282>`_)
-     * Recreate Docker/Conda environments after METbaseimage OS upgrade
-       (`#2338 <https://github.com/dtcenter/METplus/issues/2338>`_)
-
+     * Update Scientific Objective Documentation for some S2S Use Cases
+       (`#2628 <https://github.com/dtcenter/METplus/issues/2628>`_)
+     * Update Release Notes to include updating the schedule for releases
+       (`#2751 <https://github.com/dtcenter/METplus/issues/2751>`_)
+     * Add a dedication to all of the User's Guides
+       (`#2780 <https://github.com/dtcenter/METplus/issues/2780>`_)
+     * **Include information on how to use command line arguments to override single config variables**
+       (`#2814 <https://github.com/dtcenter/METplus/issues/2814>`_)
+     * Enhance the Release Guide documentation by consistently adding dropdown instructions
+       (`#2844 <https://github.com/dtcenter/METplus/issues/2844>`_)
+
+  .. dropdown:: Build, repository, and test
+
+     * Confirm 6.0.0 Docker images are auto-generated for dtcenter/metplus and metplus-analysis
+       (`#2756 <https://github.com/dtcenter/METplus/issues/2756>`_)
+     * Refine testing GitHub Action workflow dispatch functionality
+       (`#2816 <https://github.com/dtcenter/METplus/issues/2816>`_)
 
 .. _upgrade-instructions:
```

Comment:
@georgemccabe I suspect these upgrade instructions are actually for upgrading to the previous version, 6.0.0, rather than the new version, 6.1.0. @jprestop directed me to rework them for the new MET version. By way of example, here's the [MET-12.0.0 Upgrade Instructions](https://met.readthedocs.io/en/latest/Users_Guide/release-notes.html#met-version-12-0-0-upgrade-instructions) versus the ones for [MET-12.1.0-beta1](https://met.readthedocs.io/en/develop/Users_Guide/release-notes.html#met-upgrade-instructions). I defined "types" of upgrade info and added a dropdown with details about each type.

I don't think you NEED to necessarily modify the upgrade instructions for the beta1 release, but you should modify them eventually.